### ISSUE TITLE
configHandles.js error

### ISSUE THREAD

**Issue by ashinlaurel** [TVke/react-native-tailwindcss#40]
![image](https://user-images.githubusercontent.com/44382446/87876087-47610d00-c9f3-11ea-8900-fd6923cb1569.png)
I am getting the following error - 
Unable to resolve "../../../tailwind.config" from "node_modules/react-native-tailwindcss/util/configHandler.js"

Please do look into it

**Comment by TVke** [comment]
Hi @ashinlaurel 

with version of react-native-tailwind is this as there was a similar problem in an older version

greetings
Thomas

**Comment by ashinlaurel** [comment]
This occurred in version 1.1.9
And i commented the following lines in configHandler.js and this error disappeared.
![image](https://user-images.githubusercontent.com/44382446/88582238-fe9cfa00-d06b-11ea-8b21-b02bdf3c16b8.png)

**Comment by TVke** [comment]
Yes @ashinlaurel 

This is true but do you have a config file?
Because when you comment out that line it will not load a custom config file 
I suggest uncommenting that line and adding a config file (even an empty one is oké)
e.g.:

```
module.exports = {
  theme: {
    extend: {},
  },
}
```

greetings
Thomas

**Comment by ashinlaurel** [comment]
Yup that seemed to solve the problem. And it would be great if this file is created by default when this package is installed.

**Comment by TVke** [comment]
hi @ashinlaurel 

That's a good idea I have no idea how to do this but I would gladly accept a PR for this.

greetings
Thomas

**Comment by IJustDev** [comment]
Would love to work on this! A friend of mine told me it's still up to date, I will submit a pr right away.
